schema/reportdb/common/views/SystemHistoryKickstartReport.sql

```diff
@@ -0,0 +1,25 @@
+--
+-- Copyright (c) 2022 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE OR REPLACE VIEW SystemHistoryKickstartReport AS
```

Comment:
Is this about Kickstart only, or does it cover also autoyast installations? If the latter, then we should rename it to "SystemHistoryAutoinstallationReport"

Reply:
But didn't we say, that we may not want this ? I do not remember anymore the list.

Reply:
Renamed the view to SystemHistoryAutoinstallationReport. Other kickstart related reports are indeed low priority and not included.